wiki/People/Users_with_unique_titles/en.md

```diff
@@ -229,7 +242,7 @@ These players earned their titles through either a crazy gameplay achievement or
 | ![][flag_US] [BilliumMoto](https://osu.ppy.sh/users/3862471) | osu!mixer |
 | ![][flag_TW] [bossandy](https://osu.ppy.sh/users/360437) | Elite Nominator |
```

Comment:
no longer has the title (x2)

Reply:
Fixed in ad53d082337b564303d7026d5a652d211566538a